apps/urls.py

```diff
@@ -58,7 +58,7 @@
     ),
     url(r'^admin/', admin.site.urls),
     url(f'{API_PREFIX[1:]}/documents/events', documents.S3Events.as_view(), name='document-events'),
-    url(f'{API_PREFIX[1:]}/devices/status/', shipments.ListDevicesStatus.as_view(), name='devices-status'),
+    url(f'{API_PREFIX[1:]}/devices/status/', shipments.ShipmentOverviewListView.as_view(), name='devices-status'),
```

Comment:
Let's deprecate the /devices/status endpoint name. I think it makes sense for this to go in /shipments/overview. But we can keep both URLs mapped for now and communicate to frontend to change before removing the old endpoint.